server/main_server/src/main.cpp

```diff
@@ -125,6 +125,16 @@ namespace
     // when use of the logging system is safe.
     bool g_logger_initialized = false; // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
 
+    // Stores the grid configuration value of access_time.resolution_in_seconds. This
+    // global variable's only purpose is to allow the main server process to pass the
+    // configured value to the agent factory.
+    std::string g_atime_resolution_in_seconds; // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
+
+    // Stores the grid configuration value of access_time.batch_size. This global variable
+    // is used by the main server process to limit the number of atime updates that can be
+    // applied in one SQL batch update.
+    std::size_t g_atime_batch_size = 0; // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
```

Comment:
to the untrained eye... surprised to see `= 0` here...

Reply:
That will always be updated during server initialization. The admin would have to define the value as 0 in the database for it to be used by the server.

Reply:
resolving.